src/cmd/singularity/cli/keys_search.go

```diff
@@ -27,7 +27,7 @@ var KeysSearchCmd = &cobra.Command{
 	DisableFlagsInUseLine: true,
```

Comment:
Github won't let me comment on the correct line, but think this command should be `cobra.ExactArgs(1)` also.